File: src/classes/api/req_res/get-lantanas-response.ts
// ˅
'use strict';

import { Lantana } from '@/classes/datas/lantana';
import { GkillAPIResponse } from '../gkill-api-response';

// ˄

export class GetLantanasResponse extends GkillAPIResponse {
    // ˅
    
    // ˄

    lantanas: Array<Lantana>;

    constructor() {
        // ˅
        super()
        this.lantanas = new Array<Lantana>()
        // ˄
    }

    // ˅
    
    // ˄
}

// ˅

// ˄

File: src/app/gkill/api/req_res/get_lantanas_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/app/gkill/api/message"
	"github.com/mt3hr/gkill/src/app/gkill/dao/reps"
)

type GetLantanasResponse struct {
	Messages []*message.GkillMessage `json:"messages"`

	Errors []*message.GkillError `json:"errors"`

	Lantanas []*reps.Lantana `json:"lantanas"`
}
